pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -203,10 +235,78 @@ func (o *MustGatherOptions) completeImages() error {
 		}
 		o.Images = append(o.Images, image)
 	}
+	if o.AllImages {
+		// find all csvs and clusteroperators with the annotation "operators.openshift.io/must-gather-image"
+		var pluginImages = map[string]struct{}{}
+
+		images, err := o.annotatedCSVs()
+		if err != nil {
+			return err
+		}
+		for k := range images {
+			pluginImages[k] = images[k]
+		}
+
+		images, err = o.annotatedCOs()
+		if err != nil {
+			return err
+		}
+		for k := range images {
+			pluginImages[k] = images[k]
+		}
+
+		for i := range pluginImages {
+			o.Images = append(o.Images, i)
+		}
+	}
 	o.log("Using must-gather plug-in image: %s", strings.Join(o.Images, ", "))
 	return nil
 }
 
+func (o *MustGatherOptions) annotatedCSVs() (map[string]struct{}, error) {
+	var annotation = "operators.openshift.io/must-gather-image"
```

Comment:
can be moved into `const` group at the beginning of the file.

Reply:
Yeah, I had it as a const earlier. But then modified some commits and it got lost in rebases. :)

Reply:
Done